microsoftbotframework/conversationoperations.py

```diff
@@ -92,7 +92,7 @@ def send(self):
 
         response_url = self.urljoin(self.serviceUrl, "/v3/conversations")
 
-        response = self._request(response_url, 'post', request_json)
+        response = self._request(response_url, 'post', request_json, timeout_seconds)
```

Comment:
Please remove the timeout_seconds argument, it must be passed in at class instantiation per your other additions